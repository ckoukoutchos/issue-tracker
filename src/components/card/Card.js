import React from 'react';
import styles from './Card.module.css';

const Card = ({ children, selected }) => {
  return <div className={selected ? styles.selected : styles.card}>
    {children}
  </div>
}

export default Card;