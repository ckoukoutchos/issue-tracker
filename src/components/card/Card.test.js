import React from 'react';
import Card from './Card';
import renderer from 'react-test-renderer';

it('renders the Card component correctly', () => {
  const tree = renderer
    .create(<Card selected={true}>Text Here</Card>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});