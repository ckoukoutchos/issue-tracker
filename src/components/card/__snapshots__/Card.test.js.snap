// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Card component correctly 1`] = `
<div
  className="selected"
>
  Text Here
</div>
`;
